client/client.go

```diff
@@ -46,7 +46,7 @@ type Client struct {
 	Logger log.Logger
 }
 
-const defaultBaseURL = "https://library.sylabs.io"
+const defaultBaseURL = ""
```

Comment:
Is this referenced by any code?

Reply:
It is still used to initialize a local variable below, yes.  It doesn't need to be, but maybe at some point we'll want a different default.

Reply:
Reasonable to me